Remotion/ObjectBinding/Validation/BusinessObjectValidationResult.cs

```diff
@@ -170,26 +170,36 @@ private static IEnumerable<BusinessObjectValidationFailure> CreateBusinessObject
 
         if (markAsHandled)
         {
```

Comment:
```suggestion
        if (markAsHandled)
```

Since we're touching it anyway... 
Suggestion, it just looks "cleaner" to me.